Chef/Vagrantfile

```diff
@@ -37,8 +37,8 @@ Vagrant.configure('2') do |config|
   # Hyperv configuration
   config.vm.provider :hyperv do |h|
     h.vmname = 'RelativityDevVm'
-    h.cpus = 6
-    h.memory = 32000
+    h.cpus = 4

```

Comment:
Should these values also be reset to their production values?

Reply:
Updated